src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("TrueType");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("TrueType");
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
```

Comment:
1) it's allowed to use only box-sizing inside `*` selector
2) move font-family to `html` tag